crates/runtime/tests/models/s3_vectors.rs

```diff
@@ -29,74 +35,136 @@ mod search {
     use futures::StreamExt;
     use runtime::Runtime;
 
+    use crate::DEFAULT_TRACING_MODELS;
+    use crate::models::s3_vectors::get_package_delivery_dataset;
     use crate::utils::runtime_ready_check;
 
-    // S3 Vectors search test is based on the Bedrock embeddings
-    #[cfg(feature = "bedrock")]
     #[tokio::test]
     async fn s3_vectors_basic() -> Result<(), anyhow::Error> {
-        use crate::DEFAULT_TRACING_MODELS;
-
-        for env_var in [
-            "AWS_BEDROCK_KEY",
-            "AWS_BEDROCK_SECRET",
-            "AWS_S3_VECTORS_KEY",
-            "AWS_S3_VECTORS_SECRET",
-        ] {
+        for env_var in ["AWS_S3_VECTORS_KEY", "AWS_S3_VECTORS_SECRET"] {
             verify_env_secret_exists(env_var)
                 .await
                 .map_err(anyhow::Error::msg)?;
         }
 
         let _tracing = crate::init_tracing(DEFAULT_TRACING_MODELS);
 
-        // created model name is `titan-v2`
-        let titan_embeddings = create_titan_v2_embedding();
         let mut test_dataset = item_tpch_dataset_w_embeddings(
             "item",
-            "titan-v2",
+            "hf_minilm",
             Some(vec!["i_item_sk".to_string()]),
             None,
         );
 
         // Generate a unique index name for each test run
         let index_name = format!("test-index-{}", rand::random::<u8>() % 11);
 
-        test_dataset.vectors = Some(new_s3_vector_store(&index_name));
+        test_dataset.vectors = Some(new_s3_vector_store(
+            "spice-ci-tests-s3-vectors-basic",
+            &index_name,
+        ));
 
         let app = AppBuilder::new("search_app")
             .with_dataset(test_dataset)
-            .with_embedding(titan_embeddings)
+            .with_embedding(get_huggingface_embeddings(
+                "sentence-transformers/all-MiniLM-L6-v2",
+                "hf_minilm",
+            ))
             .build();
 
         let rt = start_app(app).await?;
 
         run_and_snapshot_query(
             &rt,
-            "SELECT i_item_sk, i_item_desc, round(score, 2) FROM vector_search(item, 'Patient') where i_item_sk > 5 order by score desc LIMIT 4;",
+            "SELECT i_item_sk, i_item_desc, round(score, 1) FROM vector_search(item, 'Patient') where i_item_sk > 5 order by score desc LIMIT 4;",
```

Comment:
This is added to make test more robust, ensure rounded score is the same across test runs